packages/manager/modules/pci/src/projects/project/storages/databases/database.service.js

```diff
@@ -1118,4 +1120,8 @@ export default class DatabaseService {
       )
       .then(({ data }) => data);
   }
+
+  getSupportLevel() {
+    return this.OvhApiMe.v6().supportLevel().$promise;
```

Comment:
**issue(blocking):** There's no need to perform an API call to retrieve this information. We already have it via `coreConfig.getUser()`.